firebase/.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+  "arrowParens": "always",
+  "bracketSameLine": false,
+  "bracketSpacing": false,
+  "semi": true,
+  "singleQuote": false,
+  "tabWidth": 2,
+  "useTabs": false
+}
```

Comment:
In the future, please split up reformats from normal pull requests, makes it easier to review